apps/mobile/__tests__/services/adoptions.test.ts

```diff
@@ -0,0 +1,48 @@
+import { getAllAdoptions } from "../../src/services/adoptions";
```

Comment:
No need to test this as well. Let's focus on test components and logic. To test components that fetch some info, [msw](https://mswjs.io/) to mock.